app/constants/view.ts

```diff
@@ -26,7 +26,7 @@ export const INDICATOR_BAR_HEIGHT = 38;
 export const JOIN_CALL_BAR_HEIGHT = 38;
 export const CURRENT_CALL_BAR_HEIGHT = 74;
 
-export const QUICK_OPTIONS_HEIGHT = 270;
+export const QUICK_OPTIONS_HEIGHT = 320;
```

Comment:
@enahum I changed the constant here. Personally, I wasn't bothered by a bit of extra space at the bottom of the quick actions when calls isn't enabled. But if we observe the `callsEnabled` in the header (and adjust the bottomSheet snapPoint accordingly), it will introduce another observer of callsEnabled... Your call, whatever you think is best.